### ISSUE TITLE
fill-mask ?

### ISSUE THREAD

**Issue by flutter-painter** [Helsinki-NLP/Opus-MT#90]
Is it possible to pre-train this model using fill-mask ?
Or to reuse pre-existing fill-mask such as https://huggingface.co/Davlan/afro-xlmr-large-61L ?
The goal is to pretrain low resource language on monolingual dataset

**Comment by jorgtied** [comment]
Which model are you referring to? Is this the wrong forum?

**Comment by flutter-painter** [comment]
I was refering to any translation model under opus-MT. 
Your 2nd question is only rhetorical apparently.

**Comment by jorgtied** [comment]
Not at all, I really thought that this question could be posted to the wrong thread and that you refer to one specific model (as you said "this model"). In any case, I would not know how to use the opus-mt models in the way you suggest.
